torch/_dynamo/variables/torch.py

```diff
@@ -234,21 +234,25 @@ def call_function(
                 return ConstantVariable(True, **options)
             else:
                 return ConstantVariable(False, **options)
-        elif (
-            self.value
-            in (
-                torch.is_floating_point,
-                torch.is_complex,
-            )
-            and isinstance(args[0], TensorVariable)
-            and args[0].dtype is not None
+        elif self.value in (
+            torch.is_floating_point,
+            torch.is_complex,
         ):
-            if self.value is torch.is_floating_point:
-                return ConstantVariable(args[0].dtype.is_floating_point, **options)
-            elif self.value is torch.is_complex:
-                return ConstantVariable(args[0].dtype.is_complex, **options)
+            input_arg = None
+            if args:
+                input_arg = args[0]
```

Comment:
Don't we have to test that there are not too many arguments? (This seems to be a preexisting condition)

Reply:
It's good to have, but usually we don't have unit tests for scenario that even don't pass eager mode.